news/2016-05-18-enhanced-instance.asciidoc

```diff
@@ -0,0 +1,174 @@
+---
+layout: news
+title: Experiments with enhanced version of javax.enterprise.inject.Instance
+author: Martin Kouba
+priority: 1
+change_frequency: weekly
+excerpt: Weld team is considering adding an enhanced version of javax.enterprise.inject.Instance to the Weld API. Explore the possibilities...
+desc: Experiments with enhanced version of javax.enterprise.inject.Instance
+---
+:linkattrs:
+
+`javax.enterprise.inject.Instance` is sometimes invaluable companion.
+Not only it allows to handle unsatisfied dependency gracefully, i.e. to avoid aborting container initialization unless really necessary:
```

Comment:
`Not only ...` does not fit here, it sort of calls for a second part of sentence saying `but it can also...!`
So how about: `One of its major strengths is the ability to gracefully handle unsatisfied dependencies, i.e. ...`